sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -32,9 +32,9 @@ import { WebResource } from '@azure/core-http';
 
 // @public
 export interface AccessPolicy {
-    expiry: string;
+    expiresOn: string;
```

Comment:
Should this be `expiresOn?: Date` like in the SASSignatureValues?

Reply:
Thanks for bringing this up @tg-msft, this seems to have been added ( in [ autorest ](https://github.com/Azure/autorest.typescript/issues/251) and handled in [ContainerClient](https://github.com/Azure/azure-sdk-for-js/blob/73e041bfd1afde68d2a7c7055bd3d786f14975eb/sdk/storage/storage-blob/src/ContainerClient.ts#L897) ) to deal with ISO 8601 format precision issues with Javascript.

We may be able to refactor so that we hide the autogenerated `SignedIdentifierModel` (which exposes AccessPolicy) in favor of `SignedIdentifier` (which expiresOn and startsOn are Date), this will stop exposing this autogen AccessPolicy. However, this refactoring would be riskier than this rename.

/cc @HarshaNalluru, @ramya-rao-a do you have any thoughts?

Reply:
Oh, if you have to use strings to get around date precision issues (like Python), then ignore my question.  Everything's good here.